website/pages/consul-on-kubernetes/index.tsx

```diff
@@ -230,39 +231,90 @@ export default function ConsulOnKubernetesPage() {
           ]}
         />
       </section>
-
-      {/* get started section */}
-      <section>
-        {/* card list */}
-        <DocsList
-          title="Documentation"
-          docs={[
-            {
-              icon: {
-                src: require('./images/docs/helm-icon.svg'),
-                alt: 'helm',
-              },
-              description:
-                'Consul offers an official Helm chart for quickly deploying and upgrading Consul on Kubernetes.',
-              cta: {
-                text: 'Heml Docs',
-                url: '#TODO',
-              },
-            },
-            {
-              icon: {
-                src: require('@hashicorp/mktg-logos/product/terraform/logomark/color.svg'),
-                alt: 'terraform',
-              },
-              description:
-                'Use Consul’s Terraform provider for deploying and maintaining Consul agents across both Kubernetes and non-Kubernetes environments.',
-              cta: {
-                text: 'Terraform Provider',
-                url: '#TODO',
-              },
-            },
-          ]}
-        />
+      <section className={s.getStartedWrapper}>
+        <h1 className={s.getStartedTitle}>Ways to get started</h1>
+        <div className={s.getStartedContent}>
+          <div className={s.getStartedCards}>
+            <CardList
```

Comment:
The breakpoints between `1120 --> 1225` feel a little odd to have a single row of cards w/ documentation side-by-side here...  Not a super strong opinion.

<img width="1364" alt="CleanShot 2021-09-28 at 11 25 05@2x" src="https://user-images.githubusercontent.com/2105067/135144530-231a02fe-8ec3-4337-9bf8-f82a0e83fd6e.png">

Reply:
Maybe we should `wrap` at a larger breakpoint?

Reply:
Actually, we can decrease the min in `grid-template-columns` which makes it look smoother